moveit_py/src/moveit/moveit_core/planning_scene/planning_scene.cpp

```diff
@@ -0,0 +1,405 @@
+                    R"(
+                    str: The name of the planning scene.
+                    )")
+
+      .def_property("robot_model", &planning_scene::PlanningScene::getRobotModel, nullptr,
+                    py::return_value_policy::move,
+                    R"(
+                    :py:class:`moveit_py.core.RobotModel`: The robot model associated to this planning scene.
+                    )")
+
+      .def_property("planning_frame", &planning_scene::PlanningScene::getPlanningFrame, nullptr,
+                    py::return_value_policy::move,
+                    R"(
+                    str: The frame in which planning is performed.
+                    )")
+
+      .def_property(
+          "current_state", &planning_scene::PlanningScene::getCurrentState,
+          py::overload_cast<const moveit_msgs::msg::RobotState&>(&planning_scene::PlanningScene::setCurrentState),
+          py::return_value_policy::move,
+          R"(
+                    :py:class:`moveit_py.core.RobotState`: The current state of the robot.
+                    )")
+
+      //.def_property("current_state", &planning_scene::PlanningScene::getCurrentState,
+      //              &moveit_py::bind_planning_scene::set_current_state, py::return_value_policy::move,
+      //              R"(
+      //              :py:class:`moveit_py.core.RobotState`: The current state of the robot.
+      //              )")
+
+      .def_property("planning_scene_message", &moveit_py::bind_planning_scene::get_planning_scene_msg, nullptr,
+                    py::return_value_policy::move)
+
+      .def_property("transforms", py::overload_cast<>(&planning_scene::PlanningScene::getTransforms), nullptr)
+
+      // methods
+      .def("knows_frame_transform",
+           py::overload_cast<const moveit::core::RobotState&, const std::string&>(
+               &planning_scene::PlanningScene::knowsFrameTransform, py::const_),
+           py::arg("robot_state"), py::arg("frame_id"),
+           R"(
+           Check if a transform to the frame id is known.
+           This will be known if id is a link name, an attached body id or a collision object.
+           Args:
+               robot_state (:py:class:`moveit_py.core.RobotState`): The robot state to check.
+               frame_id (str): The frame id to check.
+           Returns:
+               bool: True if the transform is known, false otherwise.
+           )")
+
+      .def("knows_frame_transform",
+           py::overload_cast<const std::string&>(&planning_scene::PlanningScene::knowsFrameTransform, py::const_),
+           py::arg("frame_id"),
+           R"(
+           Check if a transform to the frame id is known.
+           This will be known if id is a link name, an attached body id or a collision object.
+           Args:
+               robot_state (:py:class:`moveit_py.core.RobotState`): The robot state to check.
```

Comment:
I think this was copy-pasted from the previous implementation, but this one should point to a `PlanningScene` arg instead, yeah?

Reply:
Bump. This docstring should still be fixed, right? Looking again is it just that the `robot_state` arg should be removed in this one?